package.json

```diff
@@ -31,18 +31,19 @@
 		"@veramo/key-manager": "^4.2.0",
 		"@veramo/kms-local": "^4.2.0",
 		"did-resolver": "^4.0.1",
+		"dotenv": "^16.0.3",
```

Comment:
Why do you need this? Environment variables can generally just be read from environment right?